examples/graphql-kotlin/src/main/kotlin/example/armeria/server/graphql/kotlin/Main.kt

```diff
@@ -0,0 +1,56 @@
+package example.armeria.server.graphql.kotlin
+
+import com.expediagroup.graphql.generator.SchemaGeneratorConfig
+import com.expediagroup.graphql.generator.TopLevelObject
+import com.expediagroup.graphql.generator.toSchema
+import com.linecorp.armeria.server.Server
+import com.linecorp.armeria.server.ServerBuilder
+import com.linecorp.armeria.server.graphql.GraphqlService
+import org.slf4j.LoggerFactory
+import kotlin.concurrent.thread
+
+fun main() {
+    val server = newServer(8080)
+    Runtime.getRuntime().addShutdownHook(
+            thread(start = false) {
+                server.stop().join()
+                logger.info("Server has been stopped.")
+            }
+    )
+    server.start().join()
+}
+
+private fun newServer(port: Int): Server {
+    val sb = Server.builder()
+    sb.http(port)
+    configureService(sb)
+    return sb.build()
+}
+
+fun configureService(sb: ServerBuilder) {
+    //  TODO: GraphQLSchema instance cannot be injected at this time.
+    //        It is also mandatory to have at least one file for the schema.
+    sb.service("/graphql", GraphqlService.builder().configureGraphql({
+        it.schema(toSchema(
+            config = SchemaGeneratorConfig(listOf("example.armeria.server.graphql.kotlin")),
+            queries = listOf(TopLevelObject(UserQuery()))
+        ))
+    }).build())
+}
+
+data class User(val id: Int, val name: String)
+
+class UserQuery {
+
+    private val data = mapOf(
+            1 to User(1, "hero"),
+            2 to User(2, "human"),
+            3 to User(3, "droid")
+    )
+
+    fun userById(id: Int): User? {
+        return data[id]
+    }
+}
+
+private val logger = LoggerFactory.getLogger("Main")
```

Comment:
nit: How about using a fully qualified class name?

Reply:
I referenced the following code. 🤔 

https://github.com/line/armeria/blob/a5bfa4af8e7d2ae25b7f5ad59b0aad83184800b3/examples/annotated-http-service-kotlin/src/main/kotlin/example/armeria/server/annotated/kotlin/Main.kt#L65

Reply:
Ah... I didn't know that. Then, let's leave it as it is.

Reply:
How about changing both?
```java
private val log = LoggerFactory.getLogger(Main::class.java)
```